util.go

```diff
@@ -212,7 +215,13 @@ BEGIN:
 		// reflect.Type = string
 		default:
 			val = current.MapIndex(reflect.ValueOf(key))
-			namespace = namespace[endIdx+1:]
+			// If we exceeded the length of the namespace, then we're at the end
+			// of the namespace traversal, so set it to empty value to break loop
+			if endIdx+1 > len(namespace) {
```

Comment:
Is this code formatted ?

I would expect


```suggestion
			if endIdx + 1 > len(namespace) {
```

Maybe I'm wrong, but a gofmt might help